examples/acm-terraform-blog/part1/terraform/main.tf

```diff
@@ -0,0 +1,54 @@
+
+terraform {
+  required_providers {
+    google-beta = {
+      source  = "hashicorp/google-beta"
+      version = "3.73.0"
+    }
+  }
+}
+provider "google-beta" {
+  project = var.project
+  region  = var.region
+  zone    = var.zone
+}
+
+resource "random_id" "rand" {
+  byte_length = 4
+}
+
+resource "google_container_cluster" "cluster" {
```

Comment:
Is there a reason why we are not using the gke module?

Reply:
Reusing the response from the other comment - this example will accompany the blog intended for a broad audience, so we are trying to optimize for clarity and simplicity over re-usability. Hence we would like not to use additional modules if possible and stay with basic concepts.

Reply:
I'm not sure that follows - it's not like customers have to be an expert on the module just to apply the example.

Reply:
Agreed after more consideration; changed to use the module.